src/plugins/WebOfTrust/util/DelayedBackgroundJob.java

```diff
@@ -0,0 +1,26 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * A {@link BackgroundJob} that is executed after triggers have been aggregated for some delay.
+ *
+ * @author bertm
+ */
+public interface DelayedBackgroundJob extends BackgroundJob {
+    /**
+     * Triggers the execution of the job after either aggregating triggers for at most some delay
+     * specified by the implementation or as soon as the currently running job finishes, whichever
+     * comes last. Implementations must ensure that this method is safe to invoke from any thread
+     * at any moment.
+     */
+    @Override
+    public void triggerExecution();
```

Comment:
I've tried to improve the JavaDoc for the interface and all implementations at commit
f0c85256974ac9ad6f9d10b6a57715d487b74f7a commit eefbb14e76fb107d74a508c7aed0dcae1c94beee and especially commit f0c85256974ac9ad6f9d10b6a57715d487b74f7a

-> Done.